nix/mobile/ios/default.nix

```diff
@@ -27,23 +27,19 @@ let
         };
     };
 
-  selectedSources = [ status-go ];
-
-  # TARGETS
-  shell = mkShell {
-    buildInputs = [ xcodeWrapper ];
-  };
-
-in {
-  inherit xcodeWrapper;
+  selectedSources = [ fastlane status-go ];
 
   buildInputs = unique ([
+    cocoapods
+    fastlane.drv
     xcodeWrapper
     flock # used in reset-node_modules.sh
     procps
   ] ++ catAttrs "buildInputs" selectedSources);
+
   shellHook = ''
     ${status-go.shellHook}
+    ${fastlane.shellHook}
```

Comment:
This sounds like we're basically manually doing `concatString (catAttrs "shellHook" selectedSources)`

Reply:
Addressed in `nix-refactor` branch: 6a00c9a40c87c2325c9c0e900aa504d54695131f